src/main/frontend/util/property.cljs

```diff
@@ -27,8 +27,9 @@
 (defn properties-built-in?
   [properties]
   (and (seq properties)
-       (let [ks (map (comp keyword string/lower-case name) (keys properties))]
-         (every? (built-in-properties) ks))))
+       (let [ks (map (comp keyword string/lower-case name) (keys properties))
+             built-in-properties* (built-in-properties)]
+         (every? #(contains? built-in-properties* %) ks))))
```

Comment:
`(built-in-properties)` returns a set that can be used as a function like `(a-set key)`, one of my favorites in Clojure 😄

Reply:
Magic!